core/src/main/java/org/mskcc/cbio/oncokb/util/EvidenceUtils.java

```diff
@@ -109,8 +109,16 @@
     }
 
     public static Set<Evidence> getEvidenceByEvidenceTypesAndLevels(Set<EvidenceType> types, Set<LevelOfEvidence> levels) {
-        Set<Alteration> alterations = AlterationUtils.getAllAlterations();
-        List<Evidence> evidences = EvidenceUtils.getEvidence(new ArrayList<>(alterations), types, levels);
+        List<Evidence> evidences = new ArrayList<>();
+        for (Evidence evidence : CacheUtils.getAllEvidences()) {
+            if (types != null && types.size() > 0 && !types.contains(evidence.getEvidenceType())) {
```

Comment:
Can you merge these 2 if statements? Or why don't you write in this way?
if (condition) {
   evidences.add(evidence);
} else {
    continue;
}

Reply:
I think it's the same. Separating the conditions makes the code more readable.